config/routes.rb

```diff
@@ -3,4 +3,6 @@
 
   root controller: :restaurants, action: :index
   resources :restaurants
+
+  get '/add_to_cart/:id', controller: :carts, action: :add, as: :add_to_cart
```

Comment:
Avid using non RESTful routes.

Reply:
I have no idea what non RESTful routes are. Need to google it and try to understand Raouls route here :)

Reply:
That's a prudent thing to do.

Reply:
I googled and couldn't find @tochman. Can you give us a hint?